### PR TITLE
Add deployment example: ScienceMesh Federation

### PR DESCRIPTION
**Pull request by mmattel**
This PR adds a how to setup a federation using ScienceMesh/OCM.

The current rendering is visible on [staging](https://doc.staging.owncloud.com/ocis/next/depl-examples/federation/sciencemesh.html).

Note, a teaser image is in construction and will be added.

Language review welcomed.

No backport !

@tbsbdr

### CODE REVIEW

File: modules/ROOT/pages/depl-examples/federation/sciencemesh.adoc [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+* The *Invitations* screen opens:
++
+image:depl-examples/federation/science_mesh/sm_invitations.png[Invitations Screen, width=250]
+
+* In *Acccept invitations*, if not prefilled via the eMail link, enter the token and select the institution of the user who sent the invite from the dropdown field. Token and institution must match to be acceptable. Click btn:[Accept invitation] when done:
++
+image:depl-examples/federation/science_mesh/sm_user_accept_token.png[Invitation Data, width=250]
+
+* In window *Federated connections*, the setup federation is now shown and ready to share resources between the federation partners. This information is respective also shown on the inviter side of the federation:
```
Comment (phil-davis) [inline modules/ROOT/pages/depl-examples/federation/sciencemesh.adoc:91]:
```suggestion
* In window *Federated connections*, the setup federation is now shown and ready to share resources between the federation partners. This information is also shown on the inviter side of the federation:
```

File: modules/ROOT/pages/depl-examples/federation/sciencemesh.adoc [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+After the federation has been finally setup, federated users can now share resources. Note that a user must have accepted the invitaion to be selectable in the sharing dialogue.
+
+* In menu:Files App[Space or file > Sharing Icon], switch to `external` and start typing the user name. When found, select it:
++
+image:depl-examples/federation/science_mesh/sm_user_share_resource_search.png[Search Federation User, width=250]
++
+[IMPORTANT]
+====
+As rule of thumb:
```
Comment (phil-davis) [inline modules/ROOT/pages/depl-examples/federation/sciencemesh.adoc:112]:
```suggestion
As rules of thumb:
```

File: modules/ROOT/pages/depl-examples/federation/sciencemesh.adoc [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+* You cannot share your personal space.
+* You should not share files from your personal space for security reasons.
+* Only share project spaces or files inside project spaces.
```
Comment (phil-davis) [inline modules/ROOT/pages/depl-examples/federation/sciencemesh.adoc:116]:
```suggestion
* Only share project spaces or files/folders inside project spaces.
```

I suppose that a whole folder can be shared?

File: modules/ROOT/pages/depl-examples/federation/sciencemesh.adoc [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+// {empty} +
+
+toc::[]
+
+== Introduction
+
+{description} See the https://sciencemesh.io[ScienceMesh] link for more framework details.  
+
+To setup a federation using ScienceMesh, only a view steps are necessary:
```
[the remark this answers is not in the document]
Comment (mmattel) [reply modules/ROOT/pages/depl-examples/federation/sciencemesh.adoc:17]: always the same typo 🤦‍♂️

**Review by phil-davis — APPROVED** [review]

**Pull request merged**